Failed to make membrane

**Describe the bug**
Checked out  the project and tried to build it using `make membrane` and got the following error

```
installing go dependencies
Building Go Project...
adapters/grpc/auth_grpc.go:20:2: no required module provides package github.com/nitric-dev/membrane/interfaces/nitric/v1; to add it:
        go get github.com/nitric-dev/membrane/interfaces/nitric/v1
pluggable_membrane.go:24:2: no required module provides package github.com/nitric-dev/membrane/plugins/sdk; to add it:
        go get github.com/nitric-dev/membrane/plugins/sdk
make: *** [makefile:3: membrane] Error 1
````

it's missing `/sdk` inside `plugins` directory  ?

Thanks @jyecusch 

Tried your suggestion to build binary for local run and got the following error

```
└─ $ ▶ make dev-static
Generating Proto Sources
./contracts/proto: warning: directory does not exist.
./contracts/proto/*/**/*.proto: No such file or directory
make: *** [makefile:67: generate-proto] Error 1
```

--- a/README.md
+++ b/README.md
@@ -99,38 +99,11 @@ Download the Google Protobuf Compiler (standalone binary called `protoc`) from h
 make tests
 ```
 
-### Build Pluggable Membrane
-```bash
-# Build all of the Service Factory plugins
-make plugins
-```
-
-#### Building Pluggable Membrane Images
-
-##### Alpine Linux
-
-```bash
-make membrane-docker-alpine
-```
-
-##### Debian
-
-```bash
-make membrane-docker-debian
-```
-
-##### Both
-```bash
-make membrane-docker
-```
-
 ### Build Static Membranes
 
-#### Building Static Membrane
+#### AWS
 
-##### AWS
-
-###### Standard Binary
+##### Standard Binary
 
 > Linux support only - used in container images and for production.
 
@@ -138,7 +111,7 @@ make membrane-docker
 make aws-static
 ```
 
-###### Cross-platform Binary
+##### Cross-platform Binary
 
 Useful for local testing
 
@@ -146,15 +119,15 @@ Useful for local testing
 make aws-static-xp
 ```
 
-###### Container Images
+##### Container Images
 
 ```bash
 make aws-docker
 ```
 
-##### Google Cloud Platform
+#### Google Cloud Platform
 
-###### Standard Binary
+##### Standard Binary
 
 > Linux support only - used in container images and for production.
 
@@ -162,7 +135,7 @@ make aws-docker
 make gcp-static
 ```
 
-###### Cross-platform Binary
+##### Cross-platform Binary
 
 Useful for local testing
 
@@ -170,17 +143,17 @@ Useful for local testing
 make gcp-static-xp
 ```
 
-###### Container Images
+##### Container Images
 
 ```bash
 make gcp-docker
 ```
 
-##### Dev Membrane
+#### Dev Membrane
 
 > Note: the Dev Membrane should only be used for local development and testing.
 
-###### Standard Binary
+##### Standard Binary
 
 The dev binary is always cross-platform, since it doesn't need to be optimized for production deployments.
 
@@ -188,7 +161,7 @@ The dev binary is always cross-platform, since it doesn't need to be optimized f
 make dev-static
 ```
 
-###### Container Images
+##### Container Images
 
 ```bash
 make dev-docker

--- a/makefile
+++ b/makefile
@@ -1,7 +1,3 @@
-membrane: install
-	@echo Building Go Project...
-	@CGO_ENABLED=1 GOOS=linux go build -o bin/membrane pluggable_membrane.go
-
 init: install-tools
 	@echo Installing git hooks
 	@find .git/hooks -type l -exec rm {} \; && find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;
@@ -66,13 +62,13 @@ generate-proto:
 	@mkdir -p ./interfaces/
 	@protoc --go_out=./interfaces/ --go-grpc_out=./interfaces/ -I ./contracts/proto ./contracts/proto/*/**/*.proto
 
-# Build all service factory plugins
-plugins: aws-plugin gcp-plugin dev-plugin
-	@echo Done.
+# # Build all service factory plugins
+# plugins: aws-plugin gcp-plugin dev-plugin
+# 	@echo Done.
 
-test-plugins: install-tools
-	@echo Testing membrane plugins
-	@go run github.com/onsi/ginkgo/ginkgo -cover ./plugins/...
+# test-plugins: install-tools
+# 	@echo Testing membrane plugins
+# 	@go run github.com/onsi/ginkgo/ginkgo -cover ./plugins/...
 
 # Test the adapters
 test-adapters: install-tools generate-proto
@@ -94,10 +90,10 @@ aws-static-xp: generate-proto
 	@echo Building static AWS membrane
 	@CGO_ENABLED=0 go build -o bin/membrane -ldflags="-extldflags=-static" ./providers/aws/membrane.go
 
-# Service Factory Plugin for Pluggable Membrane
-aws-plugin:
-	@echo Building AWS Service Factory Plugin
-	@go build -buildmode=plugin -o lib/plugins/aws.so ./providers/aws/plugin.go
+# # Service Factory Plugin for Pluggable Membrane
+# aws-plugin:
+# 	@echo Building AWS Service Factory Plugin
+# 	@go build -buildmode=plugin -o lib/plugins/aws.so ./providers/aws/plugin.go
 
 aws-docker-static:
 	@docker build . -f ./providers/aws/aws.dockerfile -t nitricimages/membrane-aws
@@ -116,10 +112,10 @@ azure-static-xp: generate-proto
 	@echo Building static Azure membrane
 	@CGO_ENABLED=0 go build -o bin/membrane -ldflags="-extldflags=-static" ./providers/azure/membrane.go
 
-# Service Factory Plugin for Pluggable Membrane
-azure-plugin:
-	@echo Building Azure Service Factory Plugin
-	@go build -buildmode=plugin -o lib/plugins/azure.so ./providers/azure/plugin.go
+# # Service Factory Plugin for Pluggable Membrane
+# azure-plugin:
+# 	@echo Building Azure Service Factory Plugin
+# 	@go build -buildmode=plugin -o lib/plugins/azure.so ./providers/azure/plugin.go
 
 azure-docker-static:
 	@docker build . -f ./providers/azure/azure.dockerfile -t nitricimages/membrane-azure
@@ -137,10 +133,10 @@ gcp-static-xp: generate-proto
 	@echo Building static GCP membrane
 	@CGO_ENABLED=0 go build -o bin/membrane -ldflags="-extldflags=-static" ./providers/gcp/membrane.go
 
-# Service Factory Plugin for Pluggable Membrane
-gcp-plugin:
-	@echo Building GCP Service Factory Plugin
-	@go build -buildmode=plugin -o lib/plugins/gcp.so ./providers/gcp/plugin.go
+# # Service Factory Plugin for Pluggable Membrane
+# gcp-plugin:
+# 	@echo Building GCP Service Factory Plugin
+# 	@go build -buildmode=plugin -o lib/plugins/gcp.so ./providers/gcp/plugin.go
 
 gcp-docker-static:
 	@docker build . -f ./providers/gcp/gcp.dockerfile -t nitricimages/membrane-gcp
@@ -155,17 +151,19 @@ dev-static: generate-proto
 	@echo Building static Local membrane
 	@CGO_ENABLED=0 go build -o bin/membrane -ldflags="-extldflags=-static" ./providers/dev/membrane.go
 
-# Service Factory Plugin for Pluggable Membrane
-dev-plugin:
-	@echo Building Dev Service Factory Plugin
-	@go build -buildmode=plugin -o lib/plugins/dev.so ./providers/dev/plugin.go
+# # Service Factory Plugin for Pluggable Membrane
+# dev-plugin:
+# 	@echo Building Dev Service Factory Plugin
+# 	@go build -buildmode=plugin -o lib/plugins/dev.so ./providers/dev/plugin.go
 
 dev-docker-static:
 	@docker build . -f ./providers/dev/dev.dockerfile -t nitricimages/membrane-local
 
 dev-docker: dev-docker-static
 	@echo Built Local Docker Images
+# END Local Plugins
 
+# BEGIN DigitalOcean Plugins
 do-static: generate-proto
 	@CGO_ENABLED=0 go build -o bin/membrane -ldflags="-extldflags=-static" ./providers/do/membrane.go
 
@@ -174,14 +172,14 @@ do-docker-static:
 
 do-docker: do-docker-static
 	@echo Built Digital Ocean Docker Images
-# END Local Plugins
+# END DigitalOcean Plugins
 
-membrane-docker-alpine: generate-proto
-	@docker build . -f alpine.dockerfile -t nitric:membrane-alpine
-membrane-docker-debian: generate-proto
-	@docker build . -f debian.dockerfile -t nitric:membrane-debian
+# membrane-docker-alpine: generate-proto
+# 	@docker build . -f alpine.dockerfile -t nitric:membrane-alpine
+# membrane-docker-debian: generate-proto
+# 	@docker build . -f debian.dockerfile -t nitric:membrane-debian
 
-# Generate proto files locally before building docker images
-# TODO: Get alpine image generating its own sources
-membrane-docker: generate-proto membrane-docker-alpine membrane-docker-debian
-	@echo Built Docker Images
+# # Generate proto files locally before building docker images
+# # TODO: Get alpine image generating its own sources
+# membrane-docker: generate-proto membrane-docker-alpine membrane-docker-debian
+# 	@echo Built Docker Images

--- a/makefile
+++ b/makefile
@@ -66,9 +66,9 @@ generate-proto:
 # plugins: aws-plugin gcp-plugin dev-plugin
 # 	@echo Done.
 
-# test-plugins: install-tools
-# 	@echo Testing membrane plugins
-# 	@go run github.com/onsi/ginkgo/ginkgo -cover ./plugins/...
+test-plugins: install-tools
+	@echo Testing membrane plugins
+	@go run github.com/onsi/ginkgo/ginkgo -cover ./plugins/...
 
 # Test the adapters
 test-adapters: install-tools generate-proto

